cartographer/mapping_2d/sparse_pose_graph.cc

```diff
@@ -114,10 +116,14 @@ void SparsePoseGraph::AddScan(
     submap_states_.back().trajectory = submaps;
     CHECK_EQ(submap_states_.size(), submap_indices_.size());
   }
+  // ... One call before the first of insertion_submaps was finished
```

Comment:
horrible sentence, I aggree. Rephrase:
"The last time `AddScan` was called [related to the last comment] this happend:
The first of `insertion_submaps` was finished, so [continues in the next comment]"

Do you think these continued comments are bad in general? I don't have a strong opinion on this. It just seamed handy to visually encode information about the number of lines this comment refers to.